src/code/actions.js

```diff
@@ -32,12 +32,13 @@ export function changeAuthoring(authoring) {
   };
 }
 
-export function navigateToChallenge(_case, challenge) {
+export function navigateToChallenge(level, _case, challenge) {
```

Comment:
I left all `case` references alone for now, even though it should technically be `mission` now. Task for developer cleanup?